content/cucumber/configuration.md

```diff
@@ -1,13 +1,75 @@
 ---
 title: Cucumber Configuration
-subtitle: cucumber.yml, environment variables
+subtitle: cucumber.yml, type registry
```

Comment:
For Ruby, environment variables are still relevant, so maybe we should leave them in.
Overall I think the config page should be split into separate pages per language, like the Tools pages. I'll explain why in following comments.